src/mocks/FileUploadMocks.js

```diff
@@ -0,0 +1,17 @@
+import {rest} from 'msw';
+
+export const BASE_URL = 'http://localhost:8000/api/v2/';
+export const ENDPOINT = 'formio/fileupload/';
+
+export const mockFileUploadPost = rest.post(`${BASE_URL}${ENDPOINT}`, (req, res, ctx) => {
+  const body = {
+    url: `${BASE_URL}${ENDPOINT}/test`,
+    name: 'test',
+    size: 'test',
```

Comment:
this should be an integer - does not conform to the API docs: http://localhost:8000/api/v2/docs/#tag/formio/operation/formio_fileupload_create